.pre-commit-config.yaml

```diff
@@ -11,6 +11,7 @@ repos:
     - id: flake8
       args:
         - "--max-line-length=120"
+        - "--ignore=F405, E712, F403"
```

Comment:
can we provide a short description what it's ignoring as a comment? makes it easier to reason about.